manifests/extract.pp

```diff
@@ -41,25 +41,30 @@
   $strip_components=0,
   $purge=false,
   $user=undef,
-  $tar_command=$::osfamily ? {
-    'Solaris' => 'gtar',
-    default   => 'tar',
-  },
+  $tar_command=undef,
 ) {
 
+  require ::archive::params
```

Comment:
There's no resources in `params`, so `include` will do just the same as `require`.